.pre-commit-config.yaml

```diff
@@ -0,0 +1,34 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+  - id: check-yaml
```

Comment:
We should add the trailing whitespace, end of file fixer and toml checks IMO.

Reply:
All fine by me! I was just trying to keep it to what was already here (plus we have no TOML (yet 😉 ).